yscope-compose/README.md

```
@@ -0,0 +1,76 @@
+2. Untar the clp-package and the postgresql dataset.
+
+3. Replace the content of `/path/to/clp-json-package/etc/clp-config.yml` with the output of `demo-assets/init.sh generate_sample_clp_config`.
+
+4. Launch:
+
+```bash
+# You probably want to run a python 3.9 or newer virtual environment
+sbin/start-clp.sh
+```
+
+5. Compress:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)